perspective_reddit_bot/check_mod_actions.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A reddit bot to detect which actions subreddit moderators actually took."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+from datetime import datetime
+import json
+import praw
+import time
+
+from creds import creds
+
+
+def write_moderator_actions(reddit,
+                            line,
+                            id_key,
+                            timestamp_key,
+                            output_path,
+                            hours_to_wait):
+  record = json.loads(line)
+  comment = reddit.comment(record[id_key])
+  maybe_wait(record[timestamp_key], hours_to_wait)
+  record['approved'] = comment.approved
+  record['removed'] = comment.removed
+  with open(output_path, 'a') as o:
+    json.dump(record, o)
+    o.write('\n')
+
+
+def maybe_wait(timestamp, hours_to_wait):
```

Comment:
In the other module, we use the name "timestamp" to mean the UNIX seconds-since-epoch thing. Maybe call the variable `timestamp_string` instead?

Alternatively, consider having this function just be `wait_until(time)`, which just waits until the given time is past (returning immediately if it's already past). Then moving the timestamp string parsing and the adding hours_to_wait to the time would happen elsewhere. I think breaking it up into smaller functions makes sense to me.

Reply:
Good point. Refactored into a wait_until function. This makes the code much simpler, as you suggested.